windows/src/global/delphi/general/utilsystem.pas

```diff
@@ -329,11 +329,13 @@ procedure OpenContainingFolder(const FileName: string);
   sei: TShellExecuteInfo;
   s: string;
 begin
-  s := '/select,'+FileName;
+  if not FileExists(FileName)
+    then s := '"'+ExtractFilePath(FileName)+'"'
+    else s := '"/select,'+FileName+'"';
```

Comment:
If the file does not exist, the `/select` command opens Explorer to its default home window instead of the correct folder.